build.gradle.kts

```diff
@@ -43,18 +43,15 @@ repositories {
 }
 
 dependencies {
-    api(platform("com.exactpro.th2:bom:4.1.0"))
+    api(platform("com.exactpro.th2:bom:4.2.0"))
 
-    //FIXME: remove after relase
-    implementation("com.exactpro.th2:grpc-common:4.1.0-th2-2150-books-pages-3871780258-SNAPSHOT")
-    implementation("com.exactpro.th2:common:5.1.0-dev-version-5-3967999162-SNAPSHOT")
-    implementation("com.exactpro.th2:common-utils:0.0.1-book-and-page-3607472196-SNAPSHOT")
-    implementation("com.exactpro.th2:grpc-lw-data-provider:2.0.0-raw-messages-3847933308-SNAPSHOT")
+    implementation("com.exactpro.th2:common:5.1.1-dev")
```

Comment:
This version of common contains jackson-yaml dependency. Could you please upgrade common first please